.stylintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "colons": "never",
+  "groupOutputByFile": true,
```

Comment:
This is true by default right?

https://github.com/SimenB/stylint#groupoutputbyfile--default-true-true--false-